package defpackage;

/* JADX INFO: loaded from: classes2.dex */
public final class pyh extends ppd implements pqn {
    public static final pyh d;
    private static volatile pqs e;
    public int a;
    public pyg b;
    public pyf c;

    static {
        pyh pyhVar = new pyh();
        d = pyhVar;
        ppd.F(pyh.class, pyhVar);
    }

    private pyh() {
    }

    @Override // defpackage.ppd
    protected final Object a(int i, Object obj) {
        switch (i - 1) {
            case 0:
                return (byte) 1;
            case 1:
            default:
                return null;
            case 2:
                return E(d, "\u0001\u0002\u0000\u0001\u0001\u0002\u0002\u0000\u0000\u0000\u0001ဉ\u0000\u0002ဉ\u0001", new Object[]{"a", "b", "c"});
            case 3:
                return new pyh();
            case 4:
                return new poy(d);
            case 5:
                return d;
            case 6:
                pqs pozVar = e;
                if (pozVar == null) {
                    synchronized (pyh.class) {
                        pozVar = e;
                        if (pozVar == null) {
                            pozVar = new poz(d);
                            e = pozVar;
                        }
                        break;
                    }
                }
                return pozVar;
        }
    }
}
